scripting.md

```diff
@@ -201,7 +201,52 @@ CCL's binaries seem to be as fast as SBCL and nearly half the size.
 |        19948 | clisp.big      |  97% |        .0259 |
 ```
 
-<!-- ? We may have another trick  to distribute small executables: to make the fasl files executables. ?-->
+## Building a smaller binary with SBCL's core compression
+
+Building with SBCL's core compression can dramatically reduce your
+application's binary size. In our case, we passed from 120MB to 23MB,
+for a loss of a dozen milliseconds of start uptime, which was still
+under 50ms !
+
+Your SBCL must be built with core compression, see the documentation: http://www.sbcl.org/manual/#Saving-a-Core-Image
+
+Is it the case ?
+
+~~~lisp
+(find :sb-core-compression *features*)
+:SB-CORE-COMPRESSION
+~~~
+
+Yes, it is the case with this SBCL installed from Debian.
```

Comment:
You could mention here that if you build `sbcl` with `./make.sh --fancy`, it enables core-compression.